containers/jekyll/.devcontainer/base.Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM mcr.microsoft.com/vscode/devcontainers/ruby:2
+COPY library-scripts/meta.env /usr/local/etc/vscode-dev-containers
+
+# ENV Variables required by Jekyll
+ENV LANG=en_US.UTF-8 \
+    LANGUAGE=en_US:en \
+    TZ=Etc/UTC \
+    LC_ALL=en_US.UTF-8 \
+    LANG=en_US.UTF-8 \
+    LANGUAGE=en_US
+
+# Install bundler, latest jekyll, and github-pages for older jekyll
+RUN gem install bundler jekyll github-pages
+
+# [Choice] Node.js version: none, lts/*, 16, 14, 12
+ARG NODE_VERSION="none"
+RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi
+
+# [Optional] Uncomment this section to install additional OS packages.
+# RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+#     && apt-get -y install --no-install-recommends <your-package-list-here>
+
+# [Optional] Uncomment this line to install additional gems.
+# RUN gem install <your-gem-names-here>
```

Comment:
In general, we recommend folks add gems to their Gemfile and use `bundle install` to manage them.